bins/calc_metrics.py

```diff
@@ -65,23 +66,11 @@ def calc_metric(ref_dir, deg_dir, dump_dir, metrics, fs=None):
     result = defaultdict()
 
     for metric in tqdm(metrics):
-        if metric == "speaker_similarity":
-            print("Select the model to use for speaker similarity:")
-            print("(1) RawNet3")
-            print("(2) Resemblyzer")
-            model_choice = input("Enter the number of your choice: ").strip()
-
-            if model_choice not in ["1", "2"]:
-                print("Invalid choice. Exiting the program.")
-                sys.exit(1)
-
-            if model_choice == "1":
-                result[metric] = str(METRIC_FUNC[metric](ref_dir, deg_dir))
-            elif model_choice == "2":
-                similarity_score = extract_resemblyzer_similarity(
-                    deg_dir, ref_dir, dump_dir
-                )
-                result[metric] = str(similarity_score)
+        if metric in ["fad", "rawnet3_similarity"]:
```

Comment:
fad is included here, in the original commit, at line 64: https://github.com/open-mmlab/Amphion/commit/9682d0c8ec07ee75b4edd0a174dff3c79a5fb4d8#diff-4fa833e1c8dd8d05d182f8262a2cc5f727dc72a364db06f8acc5536eff3e6506
<img width="578" alt="Screenshot 2024-01-01 at 22 55 28" src="https://github.com/open-mmlab/Amphion/assets/37956777/0de96ef2-e21e-4790-8ade-285ad3f8a0cd">

Reply:
Owing to the firewalled status of huggingface.co on the Aliyun server, it went undetected because prior to testing calc_metrics.py, all parts concerning FAD had to be commented out, or else the script couldn't correctly initialize.

Reply:
@VocodexElysium Please review this.

Reply:
> @VocodexElysium Please review this.

Basically, if your internet environment does not support getting access to Hugging Face in the terminal, importing FAD will cause errors since it is trying to connect with Hugging Face. You can avoid this by setting the correct VPN environment or just downloading the necessary things yourself and then adjusting the FAD code to import the model on your computer rather than downloading from Hugging Face (I think MingXuan did this successfully). So I don't think removing FAD-related code is necessary since it only involves the internet environment and is solvable.